docs/install-modules.md

```diff
@@ -0,0 +1,27 @@
+AutoGluon is modularized into [sub-modules](https://packaging.python.org/guides/packaging-namespace-packages/) specialized for tabular, multimodal, or time series data. You can reduce the number of dependencies required by solely installing a specific sub-module via:  `pip install <submodule>`, where `<submodule>` may be one of the following options:
+
+- `autogluon.tabular` - functionality for tabular data (TabularPredictor)
+    - The default installation of `autogluon.tabular` standalone is a skeleton installation.
+    - Install via `pip install autogluon.tabular[all]` to get the same installation of tabular as via `pip install autogluon`
+    - Available optional dependencies: `lightgbm,catboost,xgboost,fastai`. These are included in `all`.
+    - Optional dependencies not included in `all`: `vowpalwabbit,imodels,skex,skl2onnx`.
+    - To run `autogluon.tabular` with only the optional LightGBM and CatBoost models for example, you can do: `pip install autogluon.tabular[lightgbm,catboost]`
+    - Experimental optional dependency: `skex`. This will speedup KNN models by 25x in training and inference on CPU. Use `pip install autogluon.tabular[all,skex]` to enable, or `pip install "scikit-learn-intelex<2021.5"` after a standard installation of AutoGluon.
```

Comment:
We can remove the `pip install "scikit-learn-intelex<2021.5"`, easier for them to just specify the autogluon pip install, which also works after a standard installation of AutoGluon.